### PR TITLE
Adding Travis-Ci Support For Arm64

### PR DESCRIPTION
**Pull request by ossdev07**
Travis-CI has added support for ARM64. Added ARM64 job in Travis-CI.

### CODE REVIEW

**Comment by ossdev07** [comment]
Only one Travis-CI job is failing and that is because it failed to download python3.7 from Travis archives. If we will re-run the railing test, it will pass.

**Comment by ceball** [comment]
@ossdev07 I have the same requests of you here as in https://github.com/pyviz-dev/pyct/pull/78: please explain why this is valuable and worth the increased build times.
 
The ARM64 jobs for param are e.g. 2.5 mins or 7.5 mins vs just 30 seconds for AMD64 jobs. Granted, there isn't a lot of activity right now, so it's not the end of the world. But note that on some holoviz projects it might be prohibitive. (By the way, what about trying ARM64 for datashader - that might be more interesting :P )

**Comment by ceball** [comment]
Also, param has a matrix of jobs; I'm sure not all jobs need an ARM64 version - maybe just pick one?

**Comment by jbednar** [comment]
I've restarted that job. 

I agree; I think we'd only want ARM tests to be enabled for one Python version (3.7, presumably), and we could add more in the unlikely case that an actual user files a bug for ARM64 on some other version of Python.

**Comment by ossdev07** [comment]
ping @ceball, this PR will increase the scope of testing by adding Travis-CI job for ARM64 architecture. As per your suggestion, I have modified the PR to enable ARM tests on only one Python version(3.7).

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +25,17 @@ jobs:
...
+      arch: amd64
+      stage: test
+      python: 3.7
+      env: TOX_ENV=py37
+      install:
+        - pip install tox
+      script:
+        - tox -e $TOX_ENV
+
+    - &default
```
Comment (ceball) [inline .travis.yml]: Duplicate (see line 27)

File: .travis.yml [hunk 2]
```diff
@@ -34,38 +45,48 @@ jobs:
         - tox -e $TOX_ENV
 
     - <<: *default
+      arch: amd64
```
Comment (ceball) [inline .travis.yml]: redundant (here and in all the subsequent blocks)

File: .travis.yml (hunk 1 above)
Comment (ossdev07) [reply .travis.yml]: Addressed the comments.

File: .travis.yml (hunk 2 above)
Comment (ossdev07) [reply .travis.yml]: Addressed the comments.

**Comment by ceball** [comment]
@ossdev07 Thanks for fixing - I think we can merge this now.

The new ARM64 job takes about 120 seconds, while the rest take about 30. If that becomes a problem in the future, we could change the ARM64 job to just be a daily cron job (param's unlikely to grow some new functionality that would need to be tested on ARM64 before being merged...).

**Comment by jbednar** [comment]
Thanks!

**Pull request merged**